include/exadg/structure/material/library/st_venant_kirchhoff.cpp

```diff
@@ -149,35 +155,40 @@ StVenantKirchhoff<dim, Number>::evaluate_stress(
 
   if(dim == 3)
   {
-    S[0][0] = f0 * E[0][0] + f1 * E[1][1] + f1 * E[2][2];
-    S[1][1] = f1 * E[0][0] + f0 * E[1][1] + f1 * E[2][2];
-    S[2][2] = f1 * E[0][0] + f1 * E[1][1] + f0 * E[2][2];
-    S[0][1] = f2 * (E[0][1] + E[1][0]);
-    S[1][2] = f2 * (E[1][2] + E[2][1]);
-    S[0][2] = f2 * (E[0][2] + E[2][0]);
-    S[1][0] = f2 * (E[0][1] + E[1][0]);
-    S[2][1] = f2 * (E[1][2] + E[2][1]);
-    S[2][0] = f2 * (E[0][2] + E[2][0]);
+    S[0][0] = f0 * strain_measure[0][0] + f1 * (strain_measure[1][1] + strain_measure[2][2]);
+    S[1][1] = f0 * strain_measure[1][1] + f1 * (strain_measure[0][0] + strain_measure[2][2]);
+    S[2][2] = f0 * strain_measure[2][2] + f1 * (strain_measure[0][0] + strain_measure[1][1]);
+    S[0][1] = f2 * (strain_measure[0][1] + strain_measure[1][0]);
+    S[1][2] = f2 * (strain_measure[1][2] + strain_measure[2][1]);
+    S[0][2] = f2 * (strain_measure[0][2] + strain_measure[2][0]);
```

Comment:
Here, I would no longer "symmetrize" inside the function, but pass a symmetric strain measure into the function.

Reply:
Now that we have the stress *and* strain measure implementation based on the deformation gradient, we *could* make use of this little trick again (the version I have currently in the code). If we want to go for the more obvious/straight-forward option (without symmetrizing), I can do that as well!